source/audioDucking.py

```diff
@@ -54,13 +54,25 @@ def __del__(self):
 def _setDuckingState(switch):
 	global _lastDuckedTime
 	with _duckingRefCountLock:
-		import gui
-		ATWindow=gui.mainFrame.GetHandle()
-		if switch:
-			oledll.oleacc.AccSetRunningUtilityState(ATWindow,ANRUS_ducking_AUDIO_ACTIVE|ANRUS_ducking_AUDIO_ACTIVE_NODUCK,ANRUS_ducking_AUDIO_ACTIVE|ANRUS_ducking_AUDIO_ACTIVE_NODUCK)
-			_lastDuckedTime=time.time()
-		else:
-			oledll.oleacc.AccSetRunningUtilityState(ATWindow,ANRUS_ducking_AUDIO_ACTIVE|ANRUS_ducking_AUDIO_ACTIVE_NODUCK,ANRUS_ducking_AUDIO_ACTIVE_NODUCK)
+		try:

```

Comment:
Not really related, it just kept tripping me up when testing locally. Hearing the error, and having to find out that it was just audio ducking. I can pull this out into a separate PR if you would like?

Reply:
Ah ok. I'm perfectly fine with having this solved in this pr, if you could update the initial description saying that it fixes something. It would also help to know where this access denied error is coming from and why you're ignoring it.